openedx/core/djangoapps/user_authn/urls_common.py

```diff
@@ -53,8 +54,23 @@
     url(r'^user_api/v1/account/password_reset/$', password_reset.PasswordResetView.as_view(),
         name="user_api_password_reset"),
 
+    # Moved from student/urls.py
```

Comment:
Because the URL doesn't include the old app, I don't think `student/urls.py` needs to be mentioned, unless I am missing something.

I do think you could add a comment like the following:
```
# password reset api view
```